### DISCUSSION TITLE
Domain move 10th October

### DISCUSSION THREAD

**Discussion by gilgongo** [jamulussoftware/jamulus#3404] [Announcements]
We are going to be moving our domain jamulus.io to Cloudflare tomorrow at around 13:00 UTC. 

We don't expect any significant disruption to the website in this, but in case we do, this is a reminder that you can download latest releases here:

https://github.com/jamulussoftware/jamulus/releases

(Scroll down past the list of heroes)

**Comment by ann0see** [comment]
Alternatively, SourceForge downloads are still available https://sourceforge.net/projects/llcon/

**Comment by gilgongo** [comment]
The move has now been completed without incident. All appears to be well.

**Reply by rdica** [reply]
Forgot to check, but was the previous configuration IPv6 as well? Glad to see its use now!

**Reply by gilgongo** [reply]
The name servers were responding on IPv6 in the previous configuration, but for IPv6, @softins what's the current situation with Jamulus?

**Reply by softins** [reply]
Not sure exactly what you mean, but from what I can see, the zone includes both IPv4 and IPv6 addresses for those names that are proxied (@, www and explorer). Their proxies will forward to just the addresses that we have defined. Explorer back-end is just IPv4 at the moment. The web page domains are CNAMEd to jamulussoftware.github.io, which resolves to both IPv4 and IPv6 for Github pages.

The other names in the zone, such as directory servers, are not proxied, but are plain DNS records and are CNAMEs that resolve just to IPv4. We will need to get IPv6 on the directories before Jamulus V4, but that is a while away yet.

**Reply by rdica** [reply]
I was referring to just the website https://jamulus.io not any other entities like explorer etc.
I know Cloudflare offers IPv6 connectivity if the property/zone in Cloudflare is configured for it.
:+1:

**Reply by gilgongo** [reply]
Ah, for the website - we haven't actually got `AAAA` records I've just noticed. Could add those I guess. EDIT: Done :-)
